devnet/devnet.go

```diff
@@ -0,0 +1,144 @@
+package devnet
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
+)
+
+func runLotusNode(ctx context.Context) {
+	var stdout, stderr bytes.Buffer
+	lotusNodeCmd := "lotus-seed genesis new localnet.json  && lotus-seed pre-seal --sector-size 2048 --num-sectors 10 && lotus-seed genesis add-miner localnet.json ~/.genesis-sectors/pre-seal-t01000.json && lotus daemon --lotus-make-genesis=dev.gen --genesis-template=localnet.json --bootstrap=false"
+
+	cmd := exec.CommandContext(ctx, "sh", "-c", lotusNodeCmd)
+	cmd.Stdout = &stdout
+	cmd.Stderr = &stderr
+	err := cmd.Start()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = cmd.Wait()
```

Comment:
nit: use CombinedOutput instead of Start/Wait with output buffers. Same result, and printing the output on error is still possible.